README.md

```diff
@@ -628,6 +637,7 @@ Common description. For a detailed description, use `tt help command` .
 -   `instances` - show enabled applications.
 -   `binaries` - show a list of installed binaries and their versions.
 -   `cluster` - manage cluster configuration.
+-   `env` - add tarantool and tt (installed using tt) to the current environment.
```

Comment:
> and tt

I'm not sure this one works.

```bash
tt install tt pr/664
tt install tarantool
. <(tt env)
```

```bash
which tarantool
/etc/tarantool/bin/tarantool
```
(ok)

```bash
which tt
/usr/bin/tt
apt remove tt -y
tt version
bash: /usr/bin/tt: No such file or directory
which tt
# empty
```
(not ok)

Reply:
before, system binaries were prioritized over installed binaries. fixed it.
```
better0fdead@better0fdead:~/tt$ which tarantool
/usr/bin/tarantool
better0fdead@better0fdead:~/tt$ . <(./tt env)
better0fdead@better0fdead:~/tt$ which tarantool
/home/better0fdead/tt/bin/tarantool
```

Reply:
Thanks for the updates